src/libsystemd/sd-bus/bus-dump.c

```diff
@@ -81,6 +86,8 @@ _public_ int sd_bus_message_dump(sd_bus_message *m, FILE *f, uint64_t flags) {
                 if (m->reply_cookie != 0)
                         fprintf(f, "  ReplyCookie=%" PRIu64, m->reply_cookie);
 
+                fprintf(f, "  Timestamp=%lu", ts);
```

Comment:
I think It is better to use `format_timestamp()`.

Reply:
I will look for format_timestamp() and update the PR.
If there are no specific requirements for the TS format I will use TIMESTAMP_US_UTC